marklogic-client-api/src/main/java/com/marklogic/client/impl/okhttp/OkHttpUtil.java

```diff
@@ -145,7 +145,7 @@ private static void configureHostnameVerifier(OkHttpClient.Builder clientBuilder
 	 * @param sslContext
 	 * @param trustManager
 	 */
-	private static void configureSocketFactory(OkHttpClient.Builder clientBuilder, SSLContext sslContext, X509TrustManager trustManager) {
+	static void configureSocketFactory(OkHttpClient.Builder clientBuilder, SSLContext sslContext, X509TrustManager trustManager) {
```

Comment:
Opened this up for package-level access so that `MarkLogicCloudAuthenticationConfigurer` can reuse it. It's also used when constructing the OkHttpClient used for talking to MarkLogic.